### PR TITLE
20.12.1 rns

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.12.1.md [hunk 1]
```diff
@@ -0,0 +1,1284 @@
+# Cortex XSOAR Content Release Notes for version 20.12.1 (226632)
+##### Published on 22 December 2020
+
+### Breaking Changes
+The following packs include breaking changes.
+
+- [ARIAPacketIntelligence Pack v2.0.0](#ariaPacketintelligence-pack-v200-partner-supported)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:7]:
```suggestion
- [ARIAPacketIntelligence Pack v2.0.0](#ariapacketintelligence-pack-v200-partner-supported)
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 2]
```diff
@@ -0,0 +1,1284 @@
...
+- [PhishTank Pack v2.0.2](#phishtank-pack-v202)
+- [Polygon Pack v1.0.2](#polygon-pack-v102-partner-supported)
+- [Pulsedive Pack v1.0.1](#pulsedive-pack-v101-community-contributed)
+- [RiskIQ Digital Footprint Pack v1.0.4](#riskiq-digital-footprint-pack-v104-partner-supported)
+- [Synapse Pack v1.0.1](#ynapse-pack-v101-community-contributed)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:12]:
```suggestion
- [Synapse Pack v1.0.1](#synapse-pack-v101-community-contributed)
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 3]
```diff
@@ -0,0 +1,1284 @@
...
+##### CertificateReputation
+Enrich and calculate the reputation of a certificate indicator.
+
+
+---
+
+### ARIAPacketIntelligence Pack v2.0.0 (Partner Supported)
+#### Integrations
+##### ARIA Packet Intelligence
+- **Breaking changes:** Update Integration to match the latest Packet Intelligence REST API. Removed **label_sia_region**, **label_sia_group** and **label_sia_name** arguments.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:252]: there are a few formats for `Breaking changes` in these rn, please finalize the format with @ShirleyDenkberg and make sure we are consistent

File: content-repo/extra-docs/releases/20.12.1.md [hunk 4]
```diff
@@ -0,0 +1,1284 @@
...
+
+
+---
+
+### New: Cryptocurrency Pack v1.0.0
+#### Indicator Fields
+##### Cryptocurrency Address Type
+
+#### Indicator Types
+**Packs/Cryptocurrency/IndicatorTypes/reputation-cryptocurrency.json**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:59]:
```suggestion
**Cryptocurrency Address**
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 5]
```diff
@@ -0,0 +1,1284 @@
...
+
+### New: Public DNS Feed Pack v1.0.0
+#### Integrations
+##### Public DNS Feed
+Fetches IP addresses from the Public DNS feed.
+
+
+---
+
+### New: Rapid Breach Response Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:141]: @glicht @anara123 do we want to highlight this one somehow in these release notes?

File: content-repo/extra-docs/releases/20.12.1.md [hunk 6]
```diff
@@ -0,0 +1,1284 @@
...
+- **Subject Alternative Names**
+
+- **Subject DN**
+Subject Distinguished Name
+- **Validity Not After**
+End of certificate validity period
+- **Validity Not Before**
+Starting of certificate validity period
+#### Indicator Types
+- **Packs/X509Certificate/IndicatorTypes/reputation-Certificate.json**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:235]:
```suggestion
- **Certificate**
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 7]
```diff
@@ -0,0 +1,1284 @@
...
+
+#### Layouts
+- ** - None**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:238]:
```suggestion
- **Certificate Indicator**
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 8]
```diff
@@ -0,0 +1,1284 @@
...
+
+### ARIAPacketIntelligence Pack v2.0.0 (Partner Supported)
+#### Integrations
+##### ARIA Packet Intelligence
+- **Breaking changes:** Update Integration to match the latest Packet Intelligence REST API. Removed **label_sia_region**, **label_sia_group** and **label_sia_name** arguments.
+- Use Remediation Configuration String (RCS) in the command to select SIA.
+
+---
+
+### AWS Feed Pack v1.1.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:257]: remove this one

File: content-repo/extra-docs/releases/20.12.1.md [hunk 9]
```diff
@@ -0,0 +1,1284 @@
...
+- **Code42 File Events**
+- **Code42 SecurityAlert ID**
+- **Code42 SecurityAlert Name**
+- **Code42 SecurityAlert State**
+
+---
+
+### Common Playbooks Pack v1.8.6
+#### Playbooks
+
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:442]:
```suggestion
##### Dedup - Generic v2
Deprecated. Use the **Dedup Generic v3** playbook instead.

##### Dedup - Generic v3
The new version of the Dedup playbook, which includes a new machine learning script, **PhishingDedupPreprocessingRule** that identifies duplicate Phishing incidents. 

```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 10]
```diff
@@ -0,0 +1,1284 @@
...
+##### Last Name
+- **Job Title**
+- **Employee Response Status**
+- **Employee Health Status**
+- **First Name**
+
+---
+
+### CrowdStrike Falcon Pack v1.2.10
+#### Incident Types
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:707]:
```suggestion
#### Incident Types
**CrowdStrike Falcon Incident** 
**CrowdStrike Falcon Detection**
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 11]
```diff
@@ -0,0 +1,1284 @@
...
+2. Start compliance search.
+3. Wait for compliance search completion.
+4. Get results of compliance search as an output.
+5. Get preview results if specified.
+
+---
+
+### Employee Offboarding Pack v1.0.5
+#### Incident Fields
+
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:766]:
```suggestion
**Offboarding Date**
Updated the entity to conform to the new schema standards.
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 12]
```diff
@@ -0,0 +1,1284 @@
...
+### Export Indicators Pack v1.0.1
+#### Integrations
+##### Export Indicators Service
+- Fixed an issue where processing an indicator without a value would raise an exception.
+- Upgraded the Docker image to demisto/teams:1.0.0.14318.
+
+---
+
+### Fastly Feed Pack v1.1.0
+#### Integrations
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:788]: remove this one

File: content-repo/extra-docs/releases/20.12.1.md [hunk 13]
```diff
@@ -0,0 +1,1284 @@
...
+
+### Infinipoint Pack v1.0.3 (Partner Supported)
+#### Integrations
+##### Infinipoint
+- Maintenance and stability enhancements.
+
+---
+
+### JSON Feed Pack v1.1.0
+#### Integrations
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:854]: remove this one

File: content-repo/extra-docs/releases/20.12.1.md [hunk 14]
```diff
@@ -0,0 +1,1284 @@
...
+### Majestic Million Feed Pack v1.0.1
+#### Integrations
+##### Majestic Million Feed
+Maintenance and stability enhancements.
+
+---
+
+### Microsoft Cloud App Security Pack v1.0.12
+#### Incident Types
+
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:912]:
```suggestion
- **Microsoft CAS Alert**
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 15]
```diff
@@ -0,0 +1,1284 @@
...
+### NTT Cyber Threat Sensor Pack v1.0.2 (Partner Supported)
+#### Incident Fields
+- **Graph Plot**
+- **FAERE Description**
+
+---
+
+### PAN-OS Pack v1.6.9
+#### Incident Fields
+
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:946]: - **Target Firewall Version** 
Updated the entity to conform to the new schema standards.

File: content-repo/extra-docs/releases/20.12.1.md [hunk 16]
```diff
@@ -0,0 +1,1284 @@
...
+##### Cortex XDR - IR
+Added mapping to the *XDR Alerts*, *XDR File Artifacts*, *XDR Network Artifacts*, *XDR Modification Time*, and *LastMirroredInTime* fields.
+
+#### Incident Fields
+- **XDR Modification Time**
+- **LastMirroredInTime**
+- **XDR manual severity**
+
+#### Indicator Fields
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:983]:
```suggestion
#### Indicator Fields
- **XDR Modification Time**
Updated the entity to conform to the new schema standards.
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 17]
```diff
@@ -0,0 +1,1284 @@
...
+
+#### Scripts
+##### XDRSyncScript
+Updated to support the **xdr-get-incident-extra-data** command enhancement in order to avoid rate limit errors from XDR.
+
+---
+
+### Pentera Pack v1.0.3 (Partner Supported)
+#### Incident Fields
+
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:1034]:
```suggestion
- **Pentera Operation Details**
Updated the entity to conform to the new schema standards.
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 18]
```diff
@@ -0,0 +1,1284 @@
...
+#### Playbooks
+##### SolarStorm and SUNBURST Hunting and Response Playbook
+- A new playbook for SolarStorm and SUNBURST Hunting and Response Playbook
+- Updated playbook description.
+- The playbook will now be available starting in Cortex XSOAR v5.5 instead of v6.0.
+
+---
+
+### Recorded Future Feed Pack v1.0.6
+#### Indicator Fields
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:1133]:
```suggestion
#### Indicator Fields
- **Recorded Future Evidence Details**
Updated the entity to conform to the new schema standards.
```

File: content-repo/extra-docs/releases/20.12.1.md [hunk 19]
```diff
@@ -0,0 +1,1284 @@
...
+- **What was the scope of the incident?**
+- **When was the problem first detected and by whom?**
+- **What are the areas that need improvement?**
+- **SANS Stage**
+
+---
+
+### SafeBreach - Breach and Attack Simulation platform Pack v1.1.3 (Partner Supported)
+#### Layouts
+
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.12.1.md:1160]:
```suggestion
- **SafeBreach Insight**
Updated the entity to conform to the new schema standards.
```

File: content-repo/extra-docs/releases/20.12.1.md (hunk 5 above)
Comment (anara123) [reply content-repo/extra-docs/releases/20.12.1.md:141]: Yes, we do. Maybe you can move it to the top

**Comment by ShirleyDenkberg** [comment]
@anara123 @Itay4 Doc review was completed.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**